packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -272,13 +272,13 @@ const helperBehavior = {
   getCollateralAllocated: ({ facets }, seat) =>
     seat.getAmountAllocated('Collateral', facets.helper.collateralBrand()),
   getRunAllocated: ({ facets }, seat) =>
-    seat.getAmountAllocated('RUN', facets.helper.debtBrand()),
+    seat.getAmountAllocated('Minted', facets.helper.debtBrand()),
 
   assertVaultHoldsNoRun: ({ state, facets }) => {
```

Comment:
s/assertVaultHoldsNoRun/assertVaultHoldsNoMinted/

What happened to the review button for "show how to change the code"?